docs/architecture.md

```diff
@@ -0,0 +1,68 @@
+## Architecture
+
+This app serves Reader Revenue messages (epics and banners) to theguardian.com.
+
+All epics and banners exist as a 'test', even if it only has a single variant.
+
+Test architecture overview diagram:
+https://docs.google.com/drawings/d/1QjcleJ00a0n4yfqz2vbjxBqaR_QIJEIhsLL4XmloZdQ/edit
+
+This app has two functions:
+1. Tell clients which test/variant to show, based on targeting data supplied by the client (e.g. country).
+2. Render each chosen epic/banner component.
+
+This means there is a two-step process for fetching an epic or banner from this service:
+
+```
```

Comment:
I think this diagram may be more confusing then helpful!

Reply:
I actually quite like it haha! How did you make it?

Reply:
http://asciiflow.com/